hack/ec-checks.sh

```diff
@@ -58,3 +58,20 @@ function all_tasks_dir {
     copy_all_task_versions "${task/*\//}" $tasks_dir
  done
 }
+
+function stepactions_dir {
+  if [[ ! -d $1 ]]; then
+    mkdir $1
+  fi
+  local d=$1
+
+  while IFS= read -r -d '' f; do
```

Comment:
Just by using bash this can be done like:
```suggestion
  shopt -s globstar
  for f in stepactions/**/*.yaml; do
```